ocsci/__init__.py

```diff
@@ -36,4 +55,38 @@ def to_dict(self):
         }
 
 
+def merge_dict(orig: dict, new: dict) -> dict:
+    """
+    Update a dict recursively.
+
```

Comment:
missing Args: in the docstring, I know you are using annotations for describing types, but we agreed in our conventions that we will use google sphinx style for docstrings. Thanks